fcm_django/models.py

```diff
@@ -373,13 +373,10 @@ def send_topic_message(
     ) -> Union[Optional[messaging.SendResponse], FirebaseError]:
```

Comment:
FirebaseError can be removed as a return type here

Reply:
@henriklindgren sure, let me quickly remove that